cluster_pack/filesystem.py

```diff
@@ -72,7 +72,10 @@ class EnhancedHdfsFile(pyarrow.HdfsFile):
 
     def __init__(self, base_hdfs_file):
         self.base_hdfs_file = base_hdfs_file
-        _expose_methods(self, base_hdfs_file, ignored=["write"])
+        _expose_methods(
+            self,
+            base_hdfs_file,
+            ignored=["write", "readline", "readlines"])

```

Comment:
I would also implement __iter__ and __next__ like in https://github.com/dask/hdfs3/blob/master/hdfs3/core.py

Reply:
Added __iter__. Not necessary to add it to expose_method because I ignored all methods starting with "__"
The value of __next__ is not clear to me. Not used for list iteration at least. The 'yield' already takes care of this. We could add a ticket to study if the comment 'Enables reading a file as a buffer in pandas' is still valid. Unless we don't know exactly what this is doing I would not add it.